src/css/globals/_all.scss

```diff
@@ -1,3 +1,20 @@
 @import './animations';
 @import './styling';
 @import './scroll';
+
+
+.site-hero-header {
+  h1 {
+    width: 880px;
+    height: 197px;
+    margin: 114px 5px 60px 1px;
```

Comment:
pls tailwind nutzen wenns geht. auch bei width, height. 

Generell aber nie fixes width height bei dynamischen komponenten festlegen (z.B. text) und so wenig wie möglich fixe höhe / breite nutzen

Reply:
Text und Web inhalte verhalten sich generell sehr dynamisch. jede anwendung sollte daher auch dynamisch funktionieren. Z.B. haben leute text größen verstellt, damit sie sie besser lesen können, andere nuzten 4k displays etc.

Reply:
und statt pixel bitte wenn rem o.ä. nutzen https://css-tricks.com/theres-more-to-the-css-rem-unit-than-font-sizing/#:~:text=The%20most%20obvious%20use%20for,classic%20nested%20em%20scaling%20problem.&text=So%2C%20all%20references%20to%20rem,a%20document%2Dwide%20CSS%20variable.